packages/strapi-design-system/src/Select/__tests__/Select.spec.js

```diff
@@ -60,7 +60,7 @@ describe('Select', () => {
 
       .c9 {
         color: #32324d;
-        display: inline-block;
+        display: block;
```

Comment:
The ellipsis text shouldn't be a block element IMO

Reply:
nb: discussed and fixed during peer programming